Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -561,7 +561,8 @@ def merge
           pkg_version = bottle_hash["formula"]["pkg_version"]
 
           path.parent.cd do
-            safe_system "git", "commit", "--no-edit", "--verbose",
+            author = "BrewTestBot <homebrew-test-bot@lists.sfconservancy.org>"
+            safe_system "git", "commit", "--no-edit", "--verbose", "--author=#{author}",
```

Comment:
This isn't correct. See the `Utils.set_git_name_email!` which sets these values based on `HOMEBREW_GIT_NAME` which `brew test-bot` sets: https://github.com/Homebrew/homebrew-test-bot/blob/355af90a55c07e079009a7862c137d65ee8db75c/lib/test_bot.rb#L348-L350.

As-is this will now override these values and set the `author` to @BrewTestBot for all uses of `brew bottle --commit` e.g. non-official taps.

Reply:
This seems like an Actions-specific bug.

Reply:
But even if maintainer would run `brew pr-pull` locally on their machine, the author of the bottle commit will still not be @BrewTestBot.

Reply:
`HOMEBREW_GITHUB_NAME` etc. should be set in `brew pr-pull` for when it calls `brew bottle` in that case. If the bottle commit is being created locally, though, I'm not sure why it's desirable to have it attributed to @BrewTestBot anyway?

Reply:
Just to keep things as they are right now, I guess. If it's not required, then okay.

Reply:
Sorry 😭

Reply:
We should probably set those variables in the homebrew/core pr-publish workflow if we haven't already. I think we have but I need to check that it works properly.

Reply:
Are we pulling those commits or are we creating the bottle commits locally?

Reply:
Creating locally.

Reply:
To be that sounds a bit like we should be attributing them to the local user, yeh.